go/bin/spec

```diff
@@ -0,0 +1,9 @@
+#!/usr/bin/env bash
+set -e
+
+bin/lint
+
+# runs go test on all subdirectories exluding the vendor folder
+# might cause issues with windows users though
```

Comment:
Actually thoughts on moving all our code into `go/src/` so that we don't need to filter out the vendor folder?